public/.htaccess

```diff
@@ -22,6 +22,7 @@ DirectoryIndex index.php
     Header always set X-Content-Type-Options "nosniff"
     Header always set X-XSS-Protection "1; mode=block"
     Header always set X-Frame-Options "DENY"
+    Header always set Expect-CT "max-age=86400, enforce, report-uri='https://secret-santa.team.report-uri.com/report'"
```

Comment:
report-uri is optional so maybe don't add it? (it should be a valid URL also).

Reply:
Sure ! :+1: